app/controllers/concerns/ignore_not_found.rb

```diff
@@ -2,6 +2,6 @@
 
 module IgnoreNotFound
   def skip_sentry_exception_types
-    ApplicationController::SKIP_SENTRY_EXCEPTION_TYPES + [Common::Exceptions::RecordNotFound]
+    SentryLogging::SKIP_SENTRY_EXCEPTION_TYPES + [Common::Exceptions::RecordNotFound]
```

Comment:
Sentry has a built-in way to do this: https://docs.sentry.io/clients/ruby/config/ -- maybe it didn't when all this code was originally written? Seems like it would be much easier to move this to Sentry config

Reply:
We could possibly move the exception types defined in the SKIP_SENTRY_EXCEPTION_TYPES constant to the config, but it looks like the method gets custom defined in other places as well

Reply:
This [issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/912) is in the backlog for updating how we tag services. Let's talk about it when the tools team is reformed.

Reply:
We could do the exclusion configuration like so, which would look like:

```rb
module IgnoreNotFound
  def skip_sentry_exception_types
    Raven.configure { |c| c.excluded_exceptions += 'Common::Exceptions::RecordNotFound' }
  end
end
```

This to me is a bit less readable, and intertwines the logic for " what errors don't we want to log to APM " logic into our " which APM system are we using " logic, whereas if we were to swap to a different service (i.e. DataDog), we'd have to move this logic to a different configuration. Alternatively, we could do something more abstract:

```rb
module IgnoreNotFound
  def skip_sentry_exception_types
    ErrorLogging::SKIP_EXCEPTION_TYPES += [Common::Exceptions::RecordNotFound]
  end
end
```

That seems a bit further out of scope, and addresses a problem we don't currently have.

@johnpaulashenfelter @jholton @LindseySaari thoughts?